docs/admin/federation/index.md

```diff
@@ -137,11 +137,11 @@ cluster.
 Assuming our YAML file is located at `/cluster1/cluster.yaml`, we can run the following command to register this cluster:
 
 ```shell
-$ kubectl create -f /cluster1/cluster.yaml --cluster=federation-cluster
+$ kubectl create -f /cluster1/cluster.yaml --context=federation-cluster
```

Comment:
I don't think that we want the "-cluster" suffix on the context value.  How about `--context=federation`?